content/engineering/incidents/playbooks/ci.md

```diff
@@ -0,0 +1,225 @@
+# CI Playbook
+
+- **Maintainers**: [DevX Team](../../enablement/dev-experience/index.md).
+- **Audience**: any software engineer, no prior infrastructure knowlegde required.
+- **TL;DR** This document sums up what to do in various scenarios that can block the CI.
+
+See also: [Continuous Integration](https://docs.sourcegraph.com/dev/background-information/continuous_integration) in our product documentation.
+
+## Prerequisites
+
+In order to handle problems with the CI, the following elements are necessary:
+
+- Have access to the `sourcegraph-ci` _project_ on Google Cloud Platform.
+  - See [#it-tech-ops](https://sourcegraph.slack.com/archives/C01CSS3TC75)
+- Have the CLI `gcloud` tool installed and have authenticated yourself.
+  - See [Gain access to the cluster](../../deployments/debugging/tutorial/index.md)
+- (Optional) Install [K9s](https://k9scli.io) for easier interactions with the _pods_.
+
+## Overview
+
+The CI is what enables us to feel confident when delivering our changes to our users, and is one of the key components enabling Sourcegraph to deliver quality software. While the DevX team is in charge of managing the CI as a tool, it is essential for every engineer to be able to unblock themselves if there is a problem in order be autonomous.
+
+This page lists common failures scenarios and provide a step by step guide to get the CI back in an operational state.
+
+## Scenarios
+
+### Build has failed on the `main` branch
+
+- Gravity: _minor_
+- Impact: that commit won't be deployed on DogFood and `sourcegraph.com` until an ulterior build passes.
+- Possible causes:
+  - The `main` branch runs additional checks compared to Pull Requests builds. So it's possible that one of those checks failed.
```

Comment:
Should note + link to stuff about how our pipeline is generated based on a variety of factors